deeptrack/scatterers.py

```diff
@@ -540,6 +540,9 @@ def __init__(
         position_objective=(0, 0),
         return_fft=False,
         coherence_length=None,
+        illumination_angle=0,
+        is_iscat=False,
+        epsilon_iscat=1,
```

Comment:
Add documentation. What does "is_iscat" mean? That you take the back scattered light? Try to name it more along the physical meaning than the specific modality